storage/ndb/rest-server2/server/test_go/internal/config/init_defaults.go

```diff
@@ -50,7 +50,7 @@ func newWithDefaults() AllConfigs {
 			BufferSize:          5 * 1024 * 1024,
 			GOMAXPROCS:          -1,
 			PreAllocatedBuffers: 32,
-			BatchMaxSize:        256,
```

Comment:
This isn't the only discrepancy. I see that `RonDB.NodeIDs` and `RonDBMetadataCluster.NodeIDs` also differ in defaults. `RonDBMetadataCluster.Mgmds` has different default logic (go says it'll copy default from RonDB). `RonDBMetadataCluster.ConnectionPoolSize` has different default, as well as `Testing.MySQL.Servers[].Port` and `Testing.MySQLMetadataCluster.Servers`.

Reply:
Opened RONDB-908 for that.